api.planx.uk/modules/team/docs.yaml

```diff
@@ -26,6 +26,10 @@ paths:
   /team/{teamId}/add-member:
     put:
       summary: Add user to team, and assign them a role
+      description: 
+        "Requires authentication via the WARP platform
```

Comment:
nit: "via a Cloudflare WARP client" might be more accurate language and better match how we talk about this in IT policy etc?